### PR TITLE
adding special funcs from LINX implementation

### PR DESCRIPTION
**Pull request by cgiovanetti**
Would be great to set up some guidelines for formatting and documentation--I don't know that putting every function in its own file is the way to go (annoying to type comb.comb() etc).  Right now the polylog depends on this structure but shouldn't be hard to fix if we want something else.

Otherwise I tried to follow the scipy.special documentation.

### CODE REVIEW

File: __pycache__/comb.cpython-311.pyc
Comment (nstarman) [inline __pycache__/comb.cpython-311.pyc:1]: `__pycache__` files shouldn't be committed. We should add a gitignore file.

**Comment by nstarman** [comment]
Sorry! Can you rebase one more time. I set up CI and CD.

**Comment by cgiovanetti** [comment]
Done, look OK?

File: .DS_Store
Comment (nstarman) [inline .DS_Store:1]: Somehow these files are sneaking through the `.gitignore` 🧐

Comment (cgiovanetti) [reply .DS_Store:1]: Tried removing them manually (hopefully they don't reappear on future commits)

**Review by nstarman — COMMENTED** [review]
LGTM! Do you want to add comparison tests to scipy in this PR or in a followup?

**Comment by cgiovanetti** [comment]
Let's do it in a future one, if I have to manually resolve another conflict or rebase again I will cry

**Comment by nstarman** [comment]
🎉

**Pull request merged**